<!doctype html>
<html>
  <head>
    <title>LGSF Dashboard</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <link rel="shortcut icon" href="https://democracyclub.org.uk/static/images/logo_icon.png">
  </head>
  <body>
    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  


  <h2 id="2023-09-29-13-14">2023-09-29</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-29 13:14:54.643912</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-29 13:14:56.836796</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:14:54] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:14:55] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:14:56] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-28-12-33">2023-09-28</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-28 12:33:14.117653</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-28 12:33:16.445906</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:33:14] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:33:15] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:33:16] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-27-14-28">2023-09-27</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-27 14:28:25.008172</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-27 14:28:27.408934</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:28:25] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[14:28:26] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:28:27] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-26-13-00">2023-09-26</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-26 13:00:34.666720</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-26 13:00:36.682187</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:00:34] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:00:35] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:00:36] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-25-12-14">2023-09-25</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-25 12:14:34.351283</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-25 12:14:36.627916</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:14:34] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[12:14:35] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:14:36] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-24-13-00">2023-09-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-24 13:00:43.275453</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-24 13:00:45.331155</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:00:43] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:00:44] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:00:45] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-23-12-11">2023-09-23</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-23 12:11:13.909064</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-23 12:11:16.236835</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:11:13] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:11:14] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:11:15] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:11:16] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-22-13-48">2023-09-22</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-22 13:48:04.238696</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-22 13:48:06.468922</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:48:04] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:48:05] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:48:06] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-21-13-04">2023-09-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>1 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-21 13:04:04.405092</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-21 13:04:06.396710</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:04:04] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:04:05] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:04:06] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-20-13-45">2023-09-20</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-20 13:45:27.353603</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-20 13:45:29.602060</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:45:27] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:45:28] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:45:29] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-19-14-19">2023-09-19</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-19 14:19:47.350364</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-19 14:19:49.837936</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:19:47] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:19:48] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:19:49] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-18-13-28">2023-09-18</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-18 13:28:41.777809</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-18 13:28:43.850807</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:28:41] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[13:28:42] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:28:43] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-17-14-01">2023-09-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-17 14:01:48.291309</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-17 14:01:50.562253</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:01:48] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:01:49] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:01:50] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-16-12-18">2023-09-16</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-16 12:18:54.663856</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-16 12:18:56.716763</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:18:54] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:18:55] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:18:56] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-15-14-45">2023-09-15</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>7 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-15 14:45:17.186577</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-15 14:45:24.871756</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:45:17] Fetching Scraper for: HUN                              handlers.py:23
[14:45:21] Begin attempting to scrape: HUN                        handlers.py:27
[14:45:23] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:45:24] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-14-14-02">2023-09-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-14 14:02:01.927773</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-14 14:02:03.991697</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:02:01] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[14:02:02] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:02:03] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-13-12-24">2023-09-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-13 12:24:07.112968</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-13 12:24:09.311296</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:24:07] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:24:08] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:24:09] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-12-12-22">2023-09-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-12 12:22:35.939019</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-12 12:22:38.234149</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:22:35] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:22:36] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:22:37] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:22:38] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-11-14-23">2023-09-11</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-11 14:23:16.385736</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-11 14:23:18.560886</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:23:16] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:23:17] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:23:18] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-10-12-52">2023-09-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-10 12:52:29.333641</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-10 12:52:31.485201</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:52:29] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[12:52:30] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:52:31] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  


      </main>
      <footer class="ds-footer">
        <div class="ds-block-centered ds-text-centered ds-stack">
          <div class="ds-cluster-center">
            <ul>
              <li><a href="/lgsf-dashboard/api/failing.json">Failing JSON</a></li>
            </ul>
          </div>
          <div class="ds-copyright">
            <a href="https://democracyclub.org.uk/">
              <img src="https://DemocracyClub.github.io/design-system/images/logo_white_text.svg" alt="Democracy Club Home" />
            </a>
            <p>Copyright © 2021 Democracy Club</p>
            <p>Community Interest Company</p>
            <p>Company No: <a href="https://beta.companieshouse.gov.uk/company/09461226">09461226</a></p>
          </div>
        </div>
      </footer>
    </div>
  </body>
